### PR TITLE
Gaea c5/f5 data transfer

### PR DESCRIPTION
**Pull request by zach1221**
## Commit Queue Requirements:

- [ ] Fill out all sections of this template.
- [ ] All sub component pull requests have been reviewed by their code managers.
- [ ] Run the full RT suite (compared to current baselines) on either Hera/Derecho/Hercules AND have committed the log to my PR branch.
- [ ] Add list of all failed regression tests in "Regression Tests" section.

### Priority
- [ ] Critical Bugfix (This PR contains a critical bug fix and should be prioritized.)
- [ ] High (This PR contains a feature or fix needed for a time-sensitive project (eg, retrospectives, implementations))
- [ ] Normal

### Subcomponent (with links)

- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [ ] FV3
- [ ] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [ ] none

### Input data
- [ ] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Regression Tests:
- [ ] No changes are expected to any regression test.
- [ ] Changes are expected to the following tests:
<details><summary>FAILED REGRESSION TESTS</summary>

</details>

### Libraries

- [ ] Not Needed
- [ ] Needed
  - [ ] Create separate issue in [JCSDA/spack-stack](https://github.com/JCSDA/spack-stack) asking for update to library. Include library name, library version.
  - [ ] Add issue link from JCSDA/spack-stack following this item 

### Testing Log:
- RDHPCS
  - [ ] Hera
  - [ ] Orion
  - [ ] Hercules
  - [ ] Jet
  - [ ] Gaea
  - [ ] Derecho
- WCOSS2
  - [ ] Dogwood/Cactus
  - [ ] Acorn
- CI
  - [ ] Completed
- opnReqTest
  - [ ] N/A
  - [ ] Log attached to comment

### CODE REVIEW

File: FV3
Comment (BrianCurtis-NOAA) [inline FV3:1]: Why are there FV3 changes here?

File: modulefiles/ufs_common.lua
Comment (BrianCurtis-NOAA) [inline modulefiles/ufs_common.lua:1]: If we're changing versions, are all machines that use ufs-common going to have these versions in stack-1.5.1 ?

File: modulefiles/ufs_gaea.intel.lua
Comment (BrianCurtis-NOAA) [inline modulefiles/ufs_gaea.intel.lua:1]: Were we sticking with stack 1.5.1 on gaea?

File: tests/logs/OpnReqTests_control_p8_hera.log
Comment (BrianCurtis-NOAA) [inline tests/logs/OpnReqTests_control_p8_hera.log:1]: Where are these ORT's coming from?

File: tests/logs/RegressionTests_acorn.log
Comment (BrianCurtis-NOAA) [inline tests/logs/RegressionTests_acorn.log:1]: Are these RT's leftover from something?

**Review by BrianCurtis-NOAA — CHANGES_REQUESTED** [review]

File: FV3
Comment (zach1221) [reply FV3:1]: Hey, Brian. I'll revert these to match ufs-develop. I'm not sure how they got picked up.

File: modulefiles/ufs_common.lua
Comment (zach1221) [reply modulefiles/ufs_common.lua:1]: No, I'm going to make it match the changes in Ratko's 2093. So they all need to be changed to 1.6.0.

File: modulefiles/ufs_gaea.intel.lua
Comment (zach1221) [reply modulefiles/ufs_gaea.intel.lua:1]: PR#2093 had 1.6.0 right now.

**Pull request closed without merging**